scripts/lib/api/specialCaseResults.ts

```diff
@@ -18,30 +18,28 @@ description: API documentation for qiskit-ibm-runtime`;
 export const PROVIDER_INDEX_META = `title: Qiskit IBM Provider API Docs
 description: API documentation for qiskit-ibm-provider`;
 
+export function transformSpecialCaseUrl(url: string): string {
+  //We use `-` rather than `_` as our delimiter.
+  return url
+    .replace(/(?<=^|\/)release_notes(?=#|$)/g, "release-notes")
```

Comment:
The comment is only relevant to release notes.

```suggestion
  return url
    // We use `-` rather than `_` as our delimiter.
    .replace(/(?<=^|\/)release_notes(?=#|$)/g, "release-notes")
```

Reply:
I moved your changes into https://github.com/Qiskit/documentation/pull/628 with you as coauthor.